### DISCUSSION TITLE
빌드후 컨트롤 UI가 움직임.

### DISCUSSION THREAD

**Discussion by mayalhc** [naverz/zepeto-studio-kor#457] [4. General]
빌드후 헨드폰 테스트 할때 UI가 화면 터치 위치로 이동 됩니다.
같은 증상이 있는분 없으세요?

**Comment by x4cf** [comment]
터치패드 컨트롤러를 사용하셔서 그렇습니다
기본 UI컨트롤러를 사용해주세요 :D

**Reply by mayalhc** [reply]
아..콘트롤러가 두가지 였네요...감사합니다..
